Sources/MongoKitten/QueryBuilder.swift

```diff
@@ -281,6 +287,15 @@ public indirect enum AQT {
     
     /// Whether nothing needs to be matched. Is always true and just a placeholder
     case nothing
+    
+    /// Whether the String value within the `key` contains this `Value` as a String.
+    case contains(key: String, val: ValueProtocol)
```

Comment:
Why would you choose `ValueProtocol` above `String` like `contains(key: String, val: String)`?

In the end you're only validating String Values.